doc/Changelog.md

```diff
@@ -2,6 +2,8 @@
 
 # git master
 
+* [10](https://github.com/BlueBrain/Keyv/pull/10):
+  Tune memcached behaviour parameters
```

Comment:
Hey, maybe I can add my two cents here, since I also asked for similar clarifications recently. What I would expect to see is something along the following lines:
- What motivated this change? E.g. was there a specific performance issue? Under which circumstances was it observed?
- How does this patch solve it? E.g. this was due to a too small cache size for the Lugano cluster, it is now optimized under x, y, z constraints.
- What is the new vs. old observed behaviour? E.g. 10x speedup for files larger than 1MB, lower latency when accessing >1000 files, etc...
Of course it doesn't have to always include all the points above. If it can be summarized in a single sentence all the better (brevity is definitely good), but "tweak behaviour" is too vague. As a user and a reviewer it just makes me suspicious: why would I upgrade to the new version not knowing what has changed? Will I risk loosing or gaining something in the change?